hijack/templates/hijack/notification.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n static %}
 
 {% block content %}
-  <link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack.min.css' %}" media="screen">
+  <script src="{% static 'hijack/hijack.min.js' %}"></script>
```

Comment:
This is a breaking change for users who wrote their own `notification.html` template (which the [docs](https://django-hijack.readthedocs.io/en/latest/customization/#notification-layout) say is a supported feature).

* I'd suggest adding `mini-css-extract-plugin` to the webpack config, so that a `hijack/static/hijack/hijack.min.css` file is created.
* What exactly is `hijack.js`'s purpose? Prior to this PR, it wasn't included in the `notification.html` template at all. Is it necessary to start including it now?

Reply:
Excellent point. The JS part is need in Django admin, but not everywhere. Besides, yes, we need to keep the original css.